src/OrchardCore.Modules/OrchardCore.Users/SecurityHeadersAttribute.cs

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Brock Allen & Dominick Baier. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See LICENSE in the project root for license information.
+
+
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Filters;
+
+namespace OrchardCore.Users
+{
+    public class SecurityHeadersAttribute : ActionFilterAttribute
```

Comment:
Where is it used?

Reply:
It has really nothing to do with the purpose of this PR. Let's remove it.